tests/stress/km/stress_tests_km.cpp

```diff
@@ -1084,6 +1091,280 @@ _print_test_control_info(const test_control_info& test_control_info)
     }
 }
 
+static void
+_clean_up_tail_call_program(const fd_t& prog_map_fd)
+{
+    // Clean up tail calls.
+    for (int index = 0; index < MAX_TAIL_CALL_CNT; index++) {
+        REQUIRE(bpf_map_update_elem(prog_map_fd, &index, &ebpf_fd_invalid, 0) == 0);
+    }
+
+    LOG_VERBOSE("{} Cleanup done for map={}", __func__, prog_map_fd);
+}
+
+static fd_t
+_set_up_tailcall_program(bpf_object* object, const std::string map_name)
```

Comment:
nit: `map_name` can be a ref, i.e., `const std::string& map_name`.

Reply:
Fixed.